test/e2e/lib/gce/instance.go

```diff
@@ -96,7 +96,7 @@ func CreateInstance(instance Instance, imageName string, imageProject string) (I
 	}
 
 	instanceRunning := false
-	for i := 0; i < 30 && !instanceRunning; i++ {
+	for i := 0; i < 15 && !instanceRunning; i++ {
```

Comment:
Can you add a comment before this line on the timeout (15*20=300sec=5min)?

Reply:
Sorry. Just changed it. WDYT?